tests/post/features/authentication.feature

```diff
@@ -12,26 +12,30 @@ Feature: Authentication is up and running
     Scenario: Reach the OpenID Config
         Given the Kubernetes API is available
         And the control-plane Ingress path '/oidc' is available
```

Comment:
If I may suggest something, we have a "Ingress-specific `Given`", which is supposed to make sure the specified path is available. This check could perform all the required checks to move forward, including the container check:

- First, check that the `nginx-ingress-control-plane` DaemonSet is OK (there's a Pod running on all `master` Nodes, or maybe at least one, not sure)
- Then, check that for each Pod belonging in this DaemonSet, all of its containers are Ready
- Then, through one of the available control plane IPs (in our case, there's only the Bootstrap's that should work), try to connect to `https://<cp-ip>:8443/<path>`, with `<path>` being parsed in the `Given`

If all of these succeed (the last check should just validate that we do not receive a 502 or a 404 from the default backend), then any following query should work. And there's no need to _know_ in advance which Pod make up the service that the Ingress object is pointing to.

Reply:
I think point 1 & 2 have been addressed in this PR maybe not word for word.

I am going to open a second PR to address point 3 separately